### PR TITLE
Add the Best Practices Team

### PR DESCRIPTION
**Pull request by DanielSiepmann**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,6 +75,74 @@ <h2>
 
                 <!-- Please keep the team pages in alphabetical order, too. -->
 
+                <!-- Best Practices Team -->
+                <section data-background-image="images/best-practices-team.webp" data-background-color="#b6ccd3">
```
Comment (DanielSiepmann) [inline index.html]: The default text colour doesn't work with every background image.
Is there a way to alter the text colour per slide?

**Comment by DanielSiepmann** [comment]
Currently, in draft as I would like to solve the text/background colour issue first.
Also waiting for feedback from team members.

File: index.html (hunk 1 above)
Comment (oliverklee) [reply index.html]: Yes:

```suggestion
                <section data-background-image="images/best-practices-team.webp" data-background-color="#b6ccd3" style="color: #fff;">
```

(You might need to set the color for the heading in the `h2` tag as it has its own styling.)

File: index.html [hunk 2]
```diff
@@ -72,6 +75,74 @@ <h2>
...
+                    </p>
+                </section>
+                <section data-background-color="#b6ccd3">
+                    <h2>
+                        Best Practices Team
+                    </h2>
+                    <h3>
+                        Our mission ("What?")
+                    </h3>
+                    <p>
```
Comment (oliverklee) [inline index.html]: Currently, with the (by default) centered text, this is a bit hard to read. Maybe we can use bullet points instead?

File: index.html (hunk 1 above)
Comment (DanielSiepmann) [reply index.html]: I guess we actually need some blur/contrast/surrounding, etc. I'm not a designer or frontend and don't know the proper names.

The image has many colours which makes it hard to find a single suiting colour.

File: index.html (hunk 2 above)
Comment (DanielSiepmann) [reply index.html]: That makes sense, I've updated the markup.

File: index.html (hunk 1 above)
Comment (oliverklee) [reply index.html]: You can surround the `h2` and `ul` with a `div` that has a semi-transparent background, e.g., like this:

```html
<div style="background-color:rgba(182, 204, 211, .8);">
```

Comment (DanielSiepmann) [reply index.html]: Thanks, that works out pretty well. Adjusted the PR and closing this discussion.

**Review by lukaszuznanski — APPROVED** [review]

**Review by oliverklee — APPROVED** [review]

**Pull request merged**